inst/HTM0/HTM0_whyUseMizer.Rmd

```diff
@@ -33,39 +35,39 @@ You can take a look at how the modelled species biomasses change through time by
 
 ```{r}
 # species' biomass through time
-plotlyBiomass(sim,start_time = 1950,end_time = 2018)
+plotlyBiomass(sim, start_time = 1950, end_time = 2018)
 # look at size spectra in final 5 years
-plotSpectra(sim,time_range = c(2015:2018),power=2)
+plotSpectra(sim, time_range = c(2015:2018), power = 2)
 ```
 
 ### Examining changes in the community relative to an unfished state 
 
 To be able to examine any changes in the community relative to an unfished state we can re-set effort = 0 and calculate the steady state. 
 
 ```{r, eval = T, echo = F, warning=F}
-sim0 <- projectToSteady(sim@params, effort = 0, return_sim = T,t_max=100)
+sim0 <- projectToSteady(params, effort = 0, return_sim = TRUE, t_max = 200)
 plotlyBiomass(sim0)
 #  or could set the initial N to this state, would be better? better to set  this up in advance so we dont have to do it here?
 ```
 
 We can compare the current  size  spectra (averaged over 2015-2019) to the unfished size spectra to assess whether there is any evidence of a size-structured trophic cascade due to fishing.
```

Comment:
I think you are not actually averaging over 2015-2019

Reply:
Yes, I removed 2019 from this as I noticed there is an NA in the effort for Dab that year... haven't had a chance to fix it.

Reply:
What I actually meant was that there is no averaging taking place in your code (or my replacement code).